polyfills/Element/prototype/classList/config.json

```diff
@@ -4,7 +4,7 @@
 		"modernizr:classlist"
 	],
 	"browsers": {
-		"ie": "6 - 9",
+		"ie": "8 - 9",
```

Comment:
I dropped support for classlist in IE < 8, because it relied on a crazy implementation from Jonathan to support object getters, which is super clever but would make the browser reallly slow.